views/templates/resultTable.tpl

```diff
@@ -103,7 +103,8 @@ requirejs.config({
     config : {
         'taoOutcomeUi/controller/resultTable' : {
             'filter' : <?=json_encode(get_data('filter'))?>,
-            'uri' : '<?=get_data("uri")?>'
+            'uri' : '<?=get_data("uri")?>',
+            'allowSqlResult': '<?=get_data("allowSqlResult")?>'
```

Comment:
if false, it will be 
`'allowSqlResult':''`

maybe better explicitly do like:
`'allowSqlResult': '<? if (get_data("allowSqlResult")) echo 'true'; else echo 'false' ?>' `

?